cmd/edenSetup.go

```diff
@@ -298,36 +296,30 @@ var setupCmd = &cobra.Command{
 						EServerPort: eServerPort,
 					}
 					files := []string{
-						"kernel", "initrd.img", "initrd.bits",
+						"kernel", "initrd.img", "initrd.bits", "installer.img", "rootfs.img",
```

Comment:
Is there any way to NOT have this explicit list and just serve up whatever comes out of `docker run lfedge/eve installer_net | tar xvf -` ?

Reply:
Good point, thank you!

Reply:
Done